lib/tasks/cron/minute.rake

```diff
@@ -0,0 +1,10 @@
+namespace :cron do
+  task minute: :log_to_stdout do
+    Rails.logger.debug 'Starting minute cron'
+
+    Rails.logger.info 'rake openstax:accounts:sync:accounts'
+    OpenStax::RescueFrom.this { Rake::Task['openstax:accounts:sync:accounts'].invoke }
```

Comment:
At first I thought this might be too often but it seems to only check for updates and seems quick.